lib/reda/containers/ERT.py

```diff
@@ -161,6 +163,52 @@ def import_bert(self, filename, **kwargs):
             print('Summary:')
             self._describe_data(data)
 
+    @append_doc_of(reda_mpt.import_das1_fd)
+    def import_mpt_fd(self, filename, **kwargs):
+        """MPT frequency domain import
+
+        timestep: int or :class:`datetime.datetime`
+            if provided use this value to set the 'timestep' column of the
+            produced dataframe. Default: 0
+
+        """
+        timestep = kwargs.get('timestep', None)
+        if 'timestep' in kwargs:
+            del (kwargs['timestep'])
+        self.logger.info('MPT DAS-1 frequency domain import')
+        with LogDataChanges(self, filter_action='import'):
+            data, electrodes, topography = reda_mpt.import_das1_fd(
+                filename, **kwargs)
+            if timestep is not None:
+                data['timestep'] = timestep
+            self._add_to_container(data)
+        if kwargs.get('verbose', False):
+            print('Summary:')
+            self._describe_data(data)
+
+    @append_doc_of(reda_mpt.import_das1_td)
+    def import_mpt_td(self, filename, **kwargs):
```

Comment:
This is related to the general restructuring, and I'm ok to leave this as is for now, but should the ERT-container be able to import polarization data? Should we remove polarization information and only import resistance data in this case?

Reply:
I'm not sure about this. Since the ERT-container has a `to_ip` method which looks for chargeability information I figured it would be ok keep it. Yet, with the current discussion about restructuring the importers I will adjust it to just import resistance information.